hyperoptimization/run_best_config.py

```diff
@@ -0,0 +1,72 @@
+import hpbandster.core.result as hpres
+from hyperoptimization.runs import from_config as run
+from hyperoptimization.workers import TVAEWorker
+from hyperoptimization.utils import parse_hyperopt_args as hyperopt
+from hyperoptimization.explore import sorted_by_value
+from argparse import ArgumentParser as Parser
+from typing import Tuple
+
+
+def experiment(parser):
+
+    parser.add_argument("dataset", help="HD5 file as expected in input by tvo.Training")
+    parser.add_argument("--Ksize", type=int, default=3, help="size of each K^n set")
+    parser.add_argument("--epochs", type=int, default=40, help="number of training epochs")
+    parser.add_argument(
+        "--net-shape",
+        required=True,
+        type=parse_net_shape,
+        help="column-separated list of layer sizes",
+    )
+    parser.add_argument("--min_lr", type=float, help="MLP min learning rate", required=True)
+    parser.add_argument("--max_lr", type=float, help="MLP max learning rate", required=True)
```

Comment:
Do I understand correctly that this method is tailored to a particular model? (similar comment above)

Reply:
As above, hyperparameter optimization requires different decisions for different parameters. E.g. a sparsity parameter is a continuous value between [0,1] but the learning rate as above has different rates. Similarly, booleans are discrete values that need different treatment.